cmd/root.go

```diff
@@ -81,6 +81,8 @@ func NewCommand() *cobra.Command {
 	cmd.PersistentFlags().BoolVar(&config.OutputValues.Enabled, "output-values", false, "inject output values into outputs (default false)")
 	cmd.PersistentFlags().StringVar(&config.OutputValues.From, "output-values-from", "", "inject output values from file into outputs (default \"\")")
 
+	cmd.PersistentFlags().BoolVar(&config.Settings.ReadComments, "read-comments", true, "use comments for variable description when description is empty")
```

Comment:
Use of `variable` here is a bit misleading, comments will be read from all the resources, i.e. variable, output, module, etc.

```suggestion
	cmd.PersistentFlags().BoolVar(&config.Settings.ReadComments, "read-comments", true, "use comments as description when description is empty")
```

Reply:
I can be wrong but my understanding is description is only applicable to input variables and output values. If so, does this look ok ?
```suggestion
	cmd.PersistentFlags().BoolVar(&config.Settings.ReadComments, "read-comments", true, "use comments for description of inputs, outputs when description is empty")
```

Reply:
You are correct, it was my bad not to clarify it more. As part of #550 we are going to expand it to all resources; i.e. modules, and resource as well.

Reply:
I see, understood. took a look at 550, appears we probably need a different flag for that. Users might want to only use comments for description of (inputs , outputs) and not extract any other comments right. Either case, I can commit your suggestion above. we probably don't have to decide right now.